### ISSUE TITLE
Select2 not accessible when using JAWS screen reader

### ISSUE THREAD

**Issue by Shilpee-arora** [FreedomScientific/VFO-standards-support#383]
### Summary
select2 (version3.5.3) is not accessible when using JAWS screenreader. User is unable to open the select2 dropdown by pressing Enter or Space. Also unable to use arrow keys to change values. Other screen readers like voiceover work fine.
> 
> Example:
> 1. Go to http://select2.github.io/select2/
> 2. Switch on Jaws.
> 3. Tab to the example of The Basics select2 and try to open it or change selection using the keyboard.
>4. Tab to the placeholder example and try opening the select2 using enter/space or arrow keys to change option.

### Expected result

> What did you expect to happen?

The user would expect enter/space to open the select2 dropdown and/or enable them to use arrow keys to change option selection.

### Actual result

> What actually did happen?
select2 doesn't respond to keyboard and is not accessible. The user is not able to open the dropdown or change options.

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...
http://select2.github.io/select2/ has a lot of examples (The Basics, Placeholder, Minimum Input, Loading Data) that demonstrate this behavior.

#### JAWS version and build number
Jaws 2020.2021.70 ILM  

 
#### Operating System and version

Windows 10 Enterprise,  version 1909

#### Browser and version:

Microsoft Edge, version 80.0.361.50

**Comment by JAWS-test** [comment]
I do not think this is a JAWS error. The select2 element is not implemented according to the ARIA specification. In the first example a button gets the focus. As a result, JAWS is not in forms mode. If I disable the virtual cursor in JAWS, I can operate the select2 element as usual. Regardless of this, JAWS incorrectly outputs the role and does not output the parent list items. There are also several minor problems, such as the redundant output of list entries. The element should therefore not be used

**Comment by JeremiahRogersVispero** [comment]
Thanks for opening this issue, @Shilpee-arora. As @JAWS-test points out, the examples on the Select2 page do not use ARIA properly. Excellent examples, along with sample HTML and JavaScript, can be found in the ARIA Authoring Practices Guide's [Combobox Pattern.](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/)
